src/background.js

```diff
@@ -5542,6 +5543,14 @@ var str = ρσ_str, repr = ρσ_repr;;
             __argnames__ : {value: ["text", "replaceCB"]}
         });
 
+        function cleanRoll(rollText) {
+            rollText = rollText.replace(/\+ \+/g, "+");
```

Comment:
Heads up, I think this implementation breaks the "display in vtt" functionality because rollText will be `0` which then causes a type error.

Seems to be fixed by just guarding against the condition:

```
if rollText == 0:
  return 0
```

But up to y'all how you'd best like to fix it.

Reply:
Yeah, it does, I've fixed it locally for now, by making it use a `.toString()` on the rollText and changed the `0` to `"0"`
Almost done converting this extension into pure javascript, then will commit the whole thing

Reply:
FYI: the conversion is done and this fix was included with it